### ISSUE TITLE
Absence of "import Dict" for generated code.

### ISSUE THREAD

**Issue by dandoh** [CSchank/petri-app-land#58]
This happens when the custom types contain "Dict".
I had to add `import Dict exposing (Dict)` in `client/src/AppName/Static/ExtraTypes.elm`
and `import Dict` in `client/src/AppName/Static/Encode.elm"

**Comment by CSchank** [comment]
Good catch, I think @necried brought this up before, but I have yet to fix it. Thanks!
